packages/node_modules/@node-red/editor-client/locales/en-US/editor.json

```diff
@@ -792,7 +792,12 @@
         "copyPath": "Copy path to item",
         "expandItems": "Expand items",
         "collapseItems": "Collapse items",
-        "duplicate": "Duplicate"
+        "duplicate": "Duplicate",
+	"array": "array",
+	"object": "object",
```

Comment:
To repeat a comment I made on here: https://github.com/node-red/node-red/pull/2361#discussion_r341782947 - I think the common data types should be provided in the catalog under `common.type.XYZ`. Can you co-ordinate with @kazuhitoyokoi so that #2361 adds the types to the catalog in the right place and then this PR can be updated to use them.

Reply:
I will commit update for message references after merge of https://github.com/node-red/node-red/pull/2361#discussion_r341782947

Reply:
#2361 has now been merged.

Reply:
thank you! updated PR to use `common.type`.